### DISCUSSION TITLE
CANNOT DOWNLOAD SOLIDITY EXTENSION IN WSL

### DISCUSSION THREAD

**Discussion by nobutakun** [Cyfrin/foundry-full-course-cu#1342] [Q&A]
i have tried to download this extension but It currently displays a notification like the picture below .  I also tried to follow the instructions but it wasn't worked 

![Screenshot 2024-01-20 000101](https://github.com/Cyfrin/foundry-full-course-f23/assets/124981015/0d6986e0-a6e1-4ada-b255-960ec81b148b)

**Comment by xyz899** [comment]
### Hi, 
Before installing on WSL try to set  your default terminal as Powershell and Install it on powershell first, restart VS Code and see if it worked. 
if not I suggest you follow the Instructions on the documentation here to install it manually for the type you want :

https://docs.soliditylang.org/en/latest/installing-solidity.html
